.ci/magic-modules/welcome-contributor.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+echo > comment/pr_comment << EOF
+Hello!  I am a robot who works on Magic Modules PRs.
+
+I have detected that you are a community contributor, so your PR will be assigned to someone with a commit-bit on this repo for initial review.  They will authorize it to run through our CI pipeline, which will generate downstream PRs.
```

Comment:
```suggestion
I have detected that you are a community contributor, so your PR will be assigned to someone with a commit-bit on this repo for initial review.  They will triage it to an appropriate primary reviewer, and authorize it to run through our CI pipeline, generating downstream PRs.
```

Reply:
Intentionally left out, but my intention there is weakly held.  I hope that the initial reviewer will do the review, and I liked setting that expectation in the message.  Sounds like you think that triage / assign is a better model?

Reply:
I think the triage part is an implementation detail. No need for contributors to be thinking about that imo.

Reply:
I like that idea too! Unfortunately, that probably won't be true some large enough percentage (30-40%?) of the time. For example- PRs against Ansible should be sent to Alex generally, Terraform PRs to `terraform-team` members etc.

Reply:
@rambleraptor - I sort of agree, but I also think that the message can also serve to set expectations for us (like with the 2 day bit I had in there), so I think Riley is right and we should include his wording, if we think we'll want to be reminded that our autoassigned job is triage, not review.  Give me a :+1: or :-1: on the suggested wording - I'm mildly inclined against and I'll submit tonight without it unless I hear otherwise.  Counting Riley's vote as "for" and mine as "against" - break the tie!

Reply:
I think the expectation should be that we're hitting the Magician button and not reviewing. After all, if the PR doesn't work, it might be too early to start triaging and doing a full review

I vote no. Sorry, @rileykarson